### ISSUE TITLE
Music often stops playing after it finishes

### ISSUE THREAD

**Issue by jon12156** [panel-attack/panel-game#22]

**Comment by LittleEndu** [comment]
I have also have it stop in the middle of the track.

Also this is a trade off between setLooping causing a segmentation fault and keeping track of the time when to play the track again. 

The fix might be loading music as static but that probably will increase the memory usage.

**Comment by sharpobject** [comment]
@LittleEndu please provide some example files that segfault when using setLooping and we can get it fixed upstream.

**Comment by LittleEndu** [comment]
All .ogg and .mp3 files cause a segfault if you play them long enough.

**Comment by sharpobject** [comment]
Ok, I will bug the love maintainers once I have an example and not bug them while I do not. It would also be helpful to know what platform and love version is affected.

**Comment by sharpobject** [comment]
Here's the main.lua I am working with atm, I'll know in an hour whether it segfaults within an hour on osx
```
local source
function love.load()
  source = love.audio.newSource("where.mp3", "stream")
  source:setLooping(true)
  source:play()
end
```

**Comment by LittleEndu** [comment]
If the mp3 is long enough that it will need segmentation it is going to segfault. I'll try something similar when I get home

**Comment by sharpobject** [comment]
No segfaults so far with https://www.burke.ro/where.mp3 or https://www.burke.ro/heyyou.mp3

**Comment by Raspberryfloof** [comment]
The segfault issue might have been a Linux-specific issue: https://bitbucket.org/rude/love/issues/1068/segfault-audionewsource-oggfileogg-with

If so, it was fixed in LOVE v0.10.0, so if we use a newer version of LOVE then that should be fixed.
